src/main/java/com/softserveinc/ita/pageobjects/ProductPage.java

```diff
@@ -26,6 +27,14 @@ public BasketModal addToCart() {
         return new BasketModal();
     }
 
+    public int getProductPrice() {
+        return parseIntPrice($x("//p[contains(@class, 'product-prices__big')]").getText());
```

Comment:
asked to take a look at a similar thing with parsing price, I have found something in CheckoutPage, check if it possible to use parseIntPrice there